src/current/v25.1/check-external-connection.md

```diff
@@ -0,0 +1,93 @@
+---
+title: CHECK EXTERNAL CONNECTION
+summary: Test the connection of each node to your cloud storage location.
+toc: true
```

Comment:
Suggest adding `docs_area: reference.sql` to the frontmatter so this matches our other pages

(PS i should probably go look if this still matters? ISTR Jesse was doing this for "reasons" but i no longer remember why)

Reply:
thanks, I filed https://cockroachlabs.atlassian.net/browse/DOC-13935 for us to remove them